### PR TITLE
Blacken, lint, test inclusive language

### PR DESCRIPTION
**Pull request by addyess**
add inclusive name check + black repo

### CODE REVIEW

**Review by stonepreston — APPROVED** [review]
LGTM

File: reactive/kubernetes_e2e.py [hunk 1]
```diff
@@ -32,121 +33,138 @@
...
+    """Remove the state set when resources are unpacked."""
     install_snaps()
 
+    # migrate to inclusive flags
+    old, new = "kubernetes-master", "kubernetes-control-plane"  # wokeignore:rule=master
+    for flag in get_flags():
+        if flag.startswith(old):
+            new_flag = flag.replace(old, new, 1)
+            clear_flag(flag)
+            set_flag(new_flag)
```
Comment (addyess) [inline reactive/kubernetes_e2e.py:52]: swapping `kubernetes-master` flags to be `kubernetes-control-plane` flags

File: reactive/kubernetes_e2e.py [hunk 2]
```diff
@@ -32,121 +33,138 @@
...
 
     missing_services = []
-    if not is_state('kubernetes-master.available'):
-        missing_services.append('kubernetes-master:http')
-    if not is_state('certificates.available'):
-        missing_services.append('certificates')
-    if not is_state('kubeconfig.ready'):
-        missing_services.append('kubernetes-master:kube-control')
+    if not is_state("kubernetes-control-plane.available"):
+        missing_services.append("kubernetes-control-plane:http")
```
Comment (addyess) [inline reactive/kubernetes_e2e.py:67]: master -> control-plane

File: reactive/kubernetes_e2e.py [hunk 3]
```diff
@@ -32,121 +33,138 @@
...
+    if not is_state("certificates.available"):
+        missing_services.append("certificates")
+    if not is_state("kubeconfig.ready"):
+        missing_services.append("kubernetes-control-plane:kube-control")
```
Comment (addyess) [inline reactive/kubernetes_e2e.py:71]: master -> control-plane

File: reactive/kubernetes_e2e.py [hunk 4]
```diff
@@ -32,121 +33,138 @@
...
+    hookenv.status_set("maintenance", "Installing kubectl snap")
+    snap.install("kubectl", channel=channel, classic=True)
+    hookenv.status_set("maintenance", "Installing kubernetes-test snap")
+    snap.install("kubernetes-test", channel=channel, classic=True)
+    set_state("kubernetes-e2e.installed")
+
+
+@when(
+    "tls_client.ca.saved",
+    "kubernetes-control-plane.available",
```
Comment (addyess) [inline reactive/kubernetes_e2e.py:107]: master -> control-plane

**Review by addyess — COMMENTED** [review]
@Cynerva can I get your feedback here?

File: reactive/kubernetes_e2e.py [hunk 5]
```diff
@@ -32,121 +32,132 @@
...
-    if not is_state('certificates.available'):
-        missing_services.append('certificates')
-    if not is_state('kubeconfig.ready'):
-        missing_services.append('kubernetes-master:kube-control')
+    if not is_state(f"{HTTP_RELATION}.available"):
+        missing_services.append(f"{APP}:{HTTP_RELATION}")
+    if not is_state("certificates.available"):
+        missing_services.append(f"{APP}:certificates")
+    if not is_state("kubeconfig.ready"):
+        missing_services.append(f"{APP}:kube-control")
```
Comment (addyess) [inline reactive/kubernetes_e2e.py:64]: if no relations are present, `juju status` will change from:

```
Missing relations: kubernetes-master:http, certificates, kubernetes-master:kube-control
```

to

```
Missing relations: kubernetes-e2e:kubernetes-master, kubernetes-e2e:certificates, kubernetes-e2e:kube-control
```

Comment (addyess) [reply reactive/kubernetes_e2e.py:64]: these reflect the three relations which need to be related to the control-plane charm and easyrsa

**Comment by mateoflorido** [comment]
We can also edit the README file to reflect the changes made for inclusive language. Could we change the charm relations from ```kubernetes-master``` to ```kubernetes-control-plane```

**Comment by addyess** [comment]
> We can also edit the README file to reflect the changes made for inclusive language. Could we change the charm relations from `kubernetes-master` to `kubernetes-control-plane`

Oh... the README.  Yes we should update the README. 
changing the name of the relation is a bit more challenging.  I asked around this concern and it seems it would take any existing installations to do this

```bash
juju remove-relation kubernetes-e2e:kubernetes-master kubernetes-control-plane::kube-api-endpoint
juju add-relation kubernetes-e2e:kubernetes-control-plane kubernetes-control-plane::kube-api-endpoint
```

this PR intends to lower the non-inclusive language, but not necessarily removing it completely in order to maintain current installation compatibility. 

I'll update the README in a second PR

**Review by mateoflorido — APPROVED** [review]
Looks good to me!

**Pull request merged**